packages/dashboard-frontend/package.json

```diff
@@ -48,6 +48,7 @@
     "inversify-inject-decorators": "^3.1.0",
     "inversify-react": "^1.1.0",
     "js-yaml": "^4.1.0",
+    "multi-ini": "^2.3.2",
```

Comment:
Please regenerate lists of used dependencies

Reply:
I did it. We reuse the same library as we have for the server-side.